draft-irtf-cfrg-opaque.md

```diff
@@ -1814,9 +1830,11 @@ between different clients that might otherwise share the same password.
 Finally, note that online guessing attacks (against any aPAKE) can be done from
 both the client side and the server side. In particular, a malicious server can
 attempt to simulate honest responses to learn the client's password.
-Implementations and deployments of OPAQUE SHOULD consider additional checks to
-mitigate this type of attack: for instance, by ensuring that there is a
-server-authenticated channel over which OPAQUE registration and login are run.
+While this constitutes an exhaustive online attack, hence as expensive as an
+online guessing attack from the client side, it is completely avoided in
+typical cases where the channel between client and server is authenticated,
+e.g., via TLS. In addition to the above, such a channel provides privacy of user
+information, including identity and envelope values.
```

Comment:
```suggestion
online guessing attack from the client side, it can be mitigated in when the channel
between client and server is authenticated, e.g., using server-authenticated TLS.
In such cases, these online attacks are limited to clients and the authenticated server
itself. Moreover, such a channel provides privacy of user information, including identity
and envelope values.
```

I don't think this statement is true. A server that authenticates over TLS can still carry out this attack. This suggestion helps clarify that authentication only changes who can carry out the attack, but does not remove it entirely.